mix.exs

```diff
@@ -32,7 +32,8 @@ defmodule Plug.Mixfile do
      {:cowboy, "~> 1.0.1 or ~> 1.1 or ~> 2.0", optional: true},
      {:ex_doc, "~> 0.17.1", only: :docs},
      {:inch_ex, ">= 0.0.0", only: :docs},
-     {:hackney, "~> 1.2.0", only: :test}]
+     {:hackney, "~> 1.2.0", only: :test},
+     {:kadabra, "~> 0.3.4", only: [:test]},]
```

Comment:
Can we change this to just be `:test` instead of `[:test]` to match the dependency above?

Reply:
I would recommend using gun for the http client so we can have a single client for http1 and http2. It also introduces less dependencies than hackney and kadabra together.

Reply:
@ericmj I think introducing gun may result in version conflicts with cowlib and ranch.

Reply:
Good point. I guess this works until we have a good client that supports both.